Update certora/README.md (#4114)
OpenZeppelin/openzeppelin-contracts: certora/README.md

# Running the certora verification tool

> These instructions detail the process for running Certora Verification Tool on the OpenZeppelin contracts.

Reviewer: ```suggestion
These instructions detail the process for running Certora Verification Tool on OpenZeppelin Contracts.
```

> Documentation for CVT and the specification language are available [here](https://certora.atlassian.net/wiki/spaces/CPD/overview)

Reviewer: ```suggestion
Documentation for CVT and the specification language are available [here](https://certora.atlassian.net/wiki/spaces/CPD/overview).
```

## Prerequisites

> Follow the [Certora installation guide](https://docs.certora.com/en/latest/docs/user-guide/getting-started/install.html) in order to get the Certora Prover Package and the `solc` executable folder in your path

Reviewer: ```suggestion
Follow the [Certora installation guide](https://docs.certora.com/en/latest/docs/user-guide/getting-started/install.html) in order to get the Certora Prover Package and the `solc` executable folder in your path.
```

> **Note**

> > An API Key is required for local testing. Although the prover will run on a Github Actions' CI environment on each Pull Request.

Reviewer: We are not running it in every pull request to avoid too high resource usage.

```suggestion
> An API Key is required for local testing. Although the prover will run on a Github Actions' CI environment on select Pull Requests.
```

Author: Wouldn't be `selected Pull Requests`?

Reviewer: Both work grammatically but `selected` seems to have the right meaning.

## Running the verification

The Certora Verification Tool proves specs for contracts, which are defined by the `./specs.json` file along with their pre-configured options.

The verification script `./run.js` is used to submit verification jobs to the Certora Verification service.

You can run it from the root of the repository with the following command:

```bash
node certora/run.js [[CONTRACT_NAME:]SPEC_NAME] [OPTIONS...]
```

Where:

> - `CONTRACT_NAME` matches the `contract` key in the `./spec.json` file and may be empty. It will run the all matching contracts if not provided.

Reviewer: ```suggestion
- `CONTRACT_NAME` matches the `contract` key in the `./spec.json` file and may be empty. It will run all matching contracts if not provided.
```
